ai/src/Makefile

```
@@ -8,7 +8,7 @@ test: venv
 	ALGORITHM="sacd" venv/bin/python3 -m pytest --timeout 30
 
 venv:
-	python3 -m venv venv
+	python3 -m venv -p 3.9 venv
```

We need to stay on v3.8 for apple metal support.
```suggestion
	python3 -m venv venv
```

---

This was meant to run on my machine that runs 3.10, reverted